lib/matplotlib/lines.py

```diff
@@ -729,7 +748,15 @@ def _is_sorted(self, x):
 
     @allow_rasterization
     def draw(self, renderer):
-        """Draw the Line with *renderer* unless visibility is *False*."""
+        """
```

Comment:
Would suggest just deleting this docstring and moving it (after adapting it) to `Artist.draw`, subclasses will then inherit it (more accurately, inspect.getdoc (which e.g. pydoc and ipython's foo? use) will lookup the docstring in the super methods).